src/Classes/ListingHydrator.php

```diff
@@ -0,0 +1,27 @@
+<?php
+namespace ListingsApp\Classes;
+use PDO;
+
+class ListingHydrator
+{
+    public static function getListing(PDO $db, string $agent_ref): Listing
+    {
+        $query= $db->prepare("SELECT `AGENT_REF` AS 'agent_ref', `ADDRESS_1` AS 'address_1', `ADDRESS_2` AS 'address_2', `TOWN` AS 'town', `POSTCODE` AS 'postcode', `DESCRIPTION` AS 'description', `BEDROOMS` AS 'bedrooms', `PRICE` AS 'price', `IMAGE` AS 'image', `TYPE` AS 'type', `STATUS` AS 'status' FROM `listings` WHERE `AGENT_REF` = 'CSL123_100259';");
```

Comment:
Why are the fields in this query caps when in getAllListings they are lowercase? I'm guessing some people have a DB with caps and some have with lowercase? Would be worth standardising this to lowercase